scripts/subscribe-lowercase-hashes.js

```diff
@@ -11,8 +11,7 @@ const getSha1 = require("../sha1-utils");
 
 
 async function subscribeLowercaseHashToHIBP(emailAddress) {
-  const lowerCasedEmail = emailAddress.toLowerCase();
-  const lowerCasedSha1 = getSha1(lowerCasedEmail);
+  const lowerCasedSha1 = getSha1.getSha1ForHIBP(emailAddress);
```

Comment:
Q: the variable name is `lowerCasedSha1`, but I believe that `getSha1ForHIBP()` returns an uppercased string.

Reply:
It's totally confusing. I think the `lowerCased` bit refers to the letters of the email address being lowercased before being encrypted.